### PR TITLE
ELEC-148: SWD Adapter Revision 1.3 

### PR DESCRIPTION
**Pull request by Grenite**
Compact the board design, added 2 pin header to act as jumper between the 3.3v line.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 *.TLT
 *.REP
 *.LOG
+altium-lib/Schematic Diagrams.SchLib
```
Comment (tituschow) [inline .gitignore]: This should not be here.

**Comment by taipingli** [comment]
Due to changes with the library, please make sure you pull the newest version and update your schematic with all the new components.

**Comment by Grenite** [comment]
The recent commits should have updated the schematic libraries from master and the .gitignore file should be corrected now.

File: MSXII_SWD_Adapter/MSXII_SWD_Adapter.PrjPcb [hunk 2]
```diff
@@ -1092,6 +1118,42 @@ NetClassSeparateForBusSections=0
 SelectedOnly=0
 UpdateVariants=1
 PartTypes=0
+ComponentLibIdentifierKind0=Library Name And Type
+ComponentLibraryIdentifier0=Midnight Sun Integrated Library.IntLib
```
Comment (tituschow) [inline MSXII_SWD_Adapter/MSXII_SWD_Adapter.PrjPcb]: Why do we still have references to this? @taipingli @KevinHQChen

**Comment by KevinHQChen** [comment]
@Grenite some small fixes:
Please update all components from SchLib, connectors P1, P2, and D1 are still using the old component naming format.
image
image
image

What's going on here?
image

This ground trace can be directly connected, no need to add an extra angle.
image

Please remove these lines from the MSXII_SWD_Adapter.PrjPcb:
`ComponentLibIdentifierKind0=Library Name And Type
ComponentLibraryIdentifier0=Midnight Sun Integrated Library.IntLib`

**Comment by tituschow** [comment]
- [ ] Remove GND trace
- [ ] Fix weird trace
- [ ] Fix silkscreen
- [ ] Replace 3v3 jumper silkscreen with 3v3 EN

**Pull request closed without merging**